common/taskTokenSerializerInterfaces.go

```diff
@@ -33,10 +33,13 @@ type (
 	TaskToken struct {
 		DomainID        string `json:"domainId"`
 		WorkflowID      string `json:"workflowId"`
+		WorkflowType    string `json:"workflowType"`
 		RunID           string `json:"runId"`
 		ScheduleID      int64  `json:"scheduleId"`
 		ScheduleAttempt int64  `json:"scheduleAttempt"`
 		ActivityID      string `json:"activityId"`
+		ActivityType    string `json:"activityType"`
+		TaskList        string `json:"taskList"`
```

Comment:
ActivityInfo already has task list. So you don't need to add it here.

Reply:
Make sense, fixed.